settings.py

```diff
@@ -98,6 +98,16 @@
     'redirect_url': 'http://olympia.dev/api/v3/accounts/authorize/',
     'scope': 'profile',
 }
+ADMIN_FXA_CONFIG = {
+    'client_id': '0f95f6474c24c1dc',
+    'client_secret':
+        'ca45e503a1b4ec9e2a3d4855d79849e098da18b7dfe42b6bc76dfed420fc1d38',
```

Comment:
It's the one for addons-frontend development. It only works on `http://localhost:3000`. We do something similar for the regular FxA setup that only works on `http://olympia.dev`.